### DISCUSSION TITLE
Covariates as Percents vs. Counts

### DISCUSSION THREAD

**Discussion by andtheWings** [ConnorDonegan/geostan#11] [Q&A] [answered]
Hi Connor! Is there any advantage in using covariates as percents vs counts. For example, the ACS-5 returns estimates of the non-Hispanic White population as counts. Since there is error in the estimate of this numerator as well as in the estimate of the denominator (total population of the census tract), it seems like converting into a percentage count might introduce more uncertainty into the model?

**Comment by ConnorDonegan** [comment, marked as answer]
Hey, I would use percent or proportions in almost every case. Usually the reasoning behind covariate choices has to do with a balance of things; raw counts can have opposite meanings in different places.  

For the MOEs/SEs for demographic proportions (if you haven't already seen this), you can go to <data.census.gov> and use the table DP05ACS DEMOGRAPHIC AND HOUSING ESTIMATES (on the data filter, go to "surveys", select ACS, then select "Data Profiles". The data profiles include percentages with MOEs for basic demographic variables. (That will give 90% MOEs; SE=MOE/1.645)

Alternatively, you can build the MOEs  or SEs yourself for proportions using the MOEs from each variable and the `tidycensus` package's `moe_prop` function <https://walker-data.com/tidycensus/reference/index.html>. This won't be the same as  what the data profile will give you, because the data profile will include sampling covariance between the two terms (so the profile is the better one, and it may show narrower MOEs).

Sometimes the estimates of demographic proportions like that are of pretty high quality

**Reply by andtheWings** [reply]
Great, I re-wrangled the data from the tables on U.S. Census so I could get better estimates of proportions and their margins of error.
